js/generator.js

```diff
@@ -96,6 +99,32 @@ class Generator {
       downloadLink.click();
     }
   }
+
+  switchTheme = () => {
+    if(this.theme === 'light') {
+      this.theme = 'dark';
+      this.switchThemeButton.style.background = 'linear-gradient(180deg, #ebedf0 0%, #ebedf0 100%)';
```

Comment:
It would be much better to define a classname for these styles in dark mode and then instead of setting styles manually, you can only toggle classnames. 🙏🏻